packages/opa-react/src/authz-provider.tsx

```diff
@@ -13,6 +9,37 @@ import {
   type BatchRequestOptions,
 } from "@styra/opa";
 import { type ServerError } from "@styra/opa/sdk/models/components";
+import { create, windowScheduler, indexedResolver } from "@yornaath/batshit";
+
+const myIndexedResolver =
```

Comment:
This is here because we don't query things by ID: we evaluate policies with paths and inputs. So when we're batching, we'll do it as follows:
Assuming a list of to-be-evaluated-requests of
```yaml
- path: foo
  input: true
- path: foo
  input: "blue"
- path: bar
  input:
    what: "ever"
```

we'll first group by path:
```yaml
foo: # path
  - input: true
  - input: "blue"
bar:
  - input:
      what: "ever"
```
and then combine the inputs to a map, using the "hash" of path+input:

```yaml
foo: # path
  h1: true
  h2: "blue"
bar:
  h3:
    what: "ever"
```
With that, we'll feed each path's sub-object into `sdk.evaluateBatch(path, ...)`, so we'll pass one array into `Promise.all()`:
- `evaluateBatch("foo", {"h1": true, "h2": "blue")`
- `evaluateBatch("bar", {"h3": { "what": "ever"})`

The results will be `{ "h1": "first", "h2": "second"}`  and `{"h3": "something third"}`, which we'll combine into a single map.

When we're retrieving the results from the cached batched results, we'll do that via `path` and `input`: construct the hash, say, `h3`, and look at up in the result map.